qa/sc_evm_test_storage_contract.py

```diff
@@ -94,7 +94,6 @@ def run_test(self):
         was_exception = False
         test_message = 'This is a message'
         try:
-            was_exception = False
             self.__set_storage_value(smart_contract, smart_contract_address, self.evm_address, test_message,
                                      static_call=True, generate_block=False)
         except EvmExecutionError as err:
```

Comment:
Consider to remove `finally` clause and `was_exception` variable and to put a fail method in the `except` clause instead.

Reply:
Done.